src/store/useRoslib.ts

```diff
@@ -11,6 +11,7 @@ const SECONDS_TO_TIMESTAMP = 1000;
 export const useRoslibStore = defineStore('roslib', () => {
   const ros = new ROSLIB.Ros({ url: undefined });
   const isWebSocketConnected = ref<boolean>(false);
+  const latency = ref<Number>(0);
```

Comment:
Small thing with typescript. Primitive types string, number and boolean are lowercase while all others you capital the first letter. Change to "number"